keras_nlp/models/bert/bert_preprocessing_test.py

```diff
@@ -35,6 +35,70 @@ def test_tokenize(self):
         self.assertAllEqual(output["segment_ids"], [0, 0, 0, 0, 0, 0, 0, 0])
         self.assertAllEqual(output["padding_mask"], [1, 1, 1, 1, 1, 1, 1, 0])
 
+    def test_tokenize_batch(self):
+        input_data = tf.constant(
+            [
+                "THE QUICK BROWN FOX.",
+                "THE QUICK BROWN FOX.",
+                "THE QUICK BROWN FOX.",
+                "THE QUICK BROWN FOX.",
+            ]
+        )
+        preprocessor = BertPreprocessor(
+            vocabulary=self.vocab,
+            sequence_length=8,
+        )
+        output = preprocessor(input_data)
+        self.assertAllEqual(output["token_ids"], [[2, 5, 6, 7, 8, 1, 3, 0]] * 4)
+        self.assertAllEqual(
+            output["segment_ids"], [[0, 0, 0, 0, 0, 0, 0, 0]] * 4
+        )
+        self.assertAllEqual(
+            output["padding_mask"], [[1, 1, 1, 1, 1, 1, 1, 0]] * 4
+        )
+
+    def test_tokenize_multiple_sentences(self):
+        sentence_one = "THE QUICK"
```

Comment:
You might want to add a docstring to each test like 
```python
"""Raw strings treated as multiple segments in the same example."""
```
Is this true only for two? What about three or more strings?

Reply:
But following on https://github.com/keras-team/keras-nlp/issues/396 we might want to change this. Let's just test it for now.